libsentrykube/tests/test_utils.py

```diff
@@ -1,17 +1,47 @@
 from pathlib import Path
-from unittest.mock import patch, MagicMock
-from libsentrykube.utils import get_service_registry_filepath, workspace_root
+import pytest
+from unittest.mock import patch
+from libsentrykube.utils import (
+    ensure_kubectl,
+    get_service_registry_filepath,
+    workspace_root,
+)
 
 
-@patch("importlib.import_module", return_value='/')
+@patch("importlib.import_module", return_value="/")
 @patch("importlib.resources.files", return_value=Path("a/b/c"))
 def test_get_service_registry_package_installed(
-    mock_import_module, 
-    mock_import_resources_path
+    mock_import_module, mock_import_resources_path
 ):
-    assert get_service_registry_filepath() == Path("a/b/c/sentry_service_registry/services.json")
+    assert get_service_registry_filepath() == Path(
+        "a/b/c/sentry_service_registry/services.json"
+    )
```

Comment:
@nikhar do we have pre-commit installed as an action to enforce we all use the same formatter with the same configuration ?

Reply:
pre-commit is what enforced this formatting, looks like @nikhars copied directly from `ops`